arguments/partialuser.js

```diff
@@ -0,0 +1,31 @@
+const { Argument } = require('klasa');
+
+module.exports = class extends Argument {
+
+	async run(arg) {
+		const user = await this.resolveUser(arg, this.client.users);
+		if (user) return user;
+		throw `Please specify a valid user.`;
```

Comment:
throw new Error('Please specify a valid user.');

Reply:
Why use that when just using "throw" makes it prompt for a valid argument?

Reply:
Whoops, good point